t/22-dashboard.t

```diff
@@ -40,6 +40,9 @@ is_deeply(\@h2, ['opensuse', 'opensuse test'], 'two groups shown (from fixtures)
 my $test_parent = $parent_groups->create({name => 'Test parent', sort_order => 2});
 
 subtest 'Validation errors' => sub {
+    $t->get_ok('/dashboard_build_results/?group=[')->status_is(400)
```

Comment:
Should we also have a test for a valid input? I can't find one.

Reply:
Right, I suppose that would make sense.